Source/Android/Bridge/Bridge.vcxproj

```diff
@@ -5,10 +5,18 @@
       <Configuration>Debug</Configuration>
       <Platform>Win32</Platform>
     </ProjectConfiguration>
+    <ProjectConfiguration Include="Debug|x64">
```

Comment:
@project64 

I believe it doesn't hurt to have it (there could eventually be an x64 Android variant).
Plus, without this change, the overall build fails for that platform.

Another alternative would be to fully leave out Android projects on x64 builds.

Which do you think is best?
In any case, we need to submit a change or builds will keep failing.